brands/amenity/police

```diff
@@ -0,0 +1,33 @@
+{
+  "amenity/police|RCMP": {
+    "countryCodes": ["ca"],
+    "tags": {
+      "amenity": "police",
+      "brand": "Royal Canadian Mounted Police",
+      "brand:wikidata": "Q335175",
+      "brand:wikipedia": "en:Royal Canadian Mounted Police",
+      "official_name": "Royal Canadian Mounted Police",
```

Comment:
Shall we stick to camelCase for property names ?

Reply:
Yes